static/js/search.js

```diff
@@ -0,0 +1,39 @@
+function log(event, data) {
```

Comment:
Avoid creating global variables and functions - especially ones with generic names like `log` and `index`.